package.json

```diff
@@ -9,18 +9,19 @@
     "lint": "eslint --ext .js,.jsx src",
     "lint:fix": "eslint --ext .js,.jsx app server --fix",
     "clean": "rimraf public && rimraf compiled",
-    "mobile": "node node_modules/react-native/local-cli/cli.js start",
+    "mobile": "node node_modules/react-native/local-cli/cli.js start --reset-cache",
     "start": "npm run build && cross-env NODE_ENV=production node compiled/server.js",
     "dev": "cross-env NODE_ENV=development nodemon",
     "build:dev": "cross-env NODE_ENV=development webpack --debug --colors --display-error-details --config ./webpack/webpack.config.js",
     "build": "npm run clean && cross-env NODE_ENV=production webpack --colors --display-error-details --config ./webpack/webpack.config.js",
     "test": "cross-env NODE_ENV=test mocha --compilers js:babel-core/register './src/**/*.spec.js'",
-    "test:watch": "cross-env NODE_ENV=test npm test -- --watch --no-single-run"
+    "test:watch": "cross-env NODE_ENV=test npm test -- --watch --no-single-run",
+    "print-schema": "babel-node scripts/print-schema.js"
```

Comment:
Maybe add a call of this script to `mobile` (maybe other) script to make it happen automatically? Then we'll able to add `data/schema.graphql` and `data/schema.json` to `.gitignore`.